src/Test/contlinear.jl

```diff
@@ -1,7 +1,7 @@
 # Continuous linear problems
 
 # Basic solver, query, resolve
-function linear1test(model::MOI.ModelLike, config::TestConfig)
+function linear1test(model::MOI.ModelLike, config::TestConfig{T}) where {T <: Number}
```

Comment:
Why adding `<: Number` here ? As `T` is a type param of `TestConfig`, it is already constrained to be `<: Number`

Reply:
No big reason; I thought maybe it adds clarity to the reader about what T is for, but I agree it’s not needed. Should I take these off?

Reply:
Yes, it's more consistent with the rest of the MOI code

Reply:
Ok, will do